README.md

```diff
@@ -25,3 +25,81 @@ Synopsis:
 ---------
 
 See the docstrings in `textgrid.py`
+
+Example:
+---------
+This is a simple example of reading a TextGrid file. 
+```
+import textgrid
+
+# Read TextGrid object from file
+tg = textgrid.TextGrid.fromFile('test.TextGrid')
+
+# Interval example
+print ("------- Interval example -------")
+print (tg[0]) # item[1]
+print (tg[0][0]) # interval[1]
+print (tg[0][0].minTime) # interval[1].xmin
+print (tg[0][0].maxTime) # interval[1].xmax
+print (tg[0][0].mark) # interval[1].mark
+
+# Point example
+print ("------- Point example -------")
+print (tg[1]) # item[2]
+print (tg[1][0]) # points[1]
+print (tg[1][0].time) # points[1].number
+print (tg[1][0].mark) # points[1].mark
+```
+
+The content of the file `test.TextGrid` is as below:
+```
+File type = "ooTextFile"
+Object class = "TextGrid"
+
+xmin = 0
+xmax = 1
+tiers? <exists>
+size = 2
+item []:
+    item [1]:
+        class = "IntervalTier"
+        name = "words"
+        xmin = 0
+        xmax = 1
+        intervals: size = 2
+        intervals [1]:
+            xmin = 0
+            xmax = 0.5
+            text = """Is anyone home?"""
+        intervals [2]:
+            xmin = 0.5
+            xmax = 1
+            text = "asked ""Pat"""
+    item [2]:
+        class = "TextTier"
+        name = "points"
+        xmin = 0
+        xmax = 1
+        points: size = 2
+        points [1]:
+            number = 0.25
+            mark = """event"""
+        points [2]:
+            number = 0.75
+            mark = """event"" with quotes again"
+```
+
+By runing the code above, the output is:
```

Comment:
"running" not "runing". But just say "the following is the output of the above snippet:" or something like that.